src/components/TextField/TextField.module.scss

```diff
@@ -326,3 +326,15 @@
 .textField .inputWrapper.wrapperSizeLarge .iconContainer .icon {
   font-family: var(--font-family);
 }
+
+input[type="date"]::-webkit-calendar-picker-indicator,
+input[type="datetime-local"]::-webkit-calendar-picker-indicator {
+  background: url("./assets/Calendar.svg") no-repeat;
+}
+
+:global(.dark-app-theme, .black-app-theme, .hacker_theme-app-theme) {
```

Comment:
Why aren’t we using a color token here that will color the SVG?

Reply:
I think those browser icons color can only be changed via `filter` or replace the svg as background